src/main/static/src/components/common/field/Field.js

```diff
@@ -113,9 +117,13 @@ class Field extends React.Component<Props, State> {
 
     getReadOnlyValue() {
         if (this.props.data) {
-            const selected = this.props.data.find(item => item.value === this.props.selectValue)
-            return selected ? selected.label : null
-    }
+            const selected = (this.props.multiselect && Array.isArray(this.props.selectValue))
+                ? this.props.data.filter(item => Array.isArray(this.props.selectValue) && this.props.selectValue.some(selectValue => selectValue === item.value))
```

Comment:
Onko tässä turha `Array.isArray(this.props.selectValue)` koska se tehdään jo edellisellä rivillä?

Reply:
Flow jostain syystä vaatii tuon. Kai siinä on jokin logiikka.